Configuration/Applications/python/ConfigBuilder.py

```diff
@@ -2160,6 +2153,11 @@ def prepare(self, doChecking = False):
 
 	self.pythonCfgCode += result
 
+        from PhysicsTools.PatAlgos.tools.helpers import schedulePatAlgosEndPath
+        schedulePatAlgosEndPath(self.process)
+        self.pythonCfgCode+="from PhysicsTools.PatAlgos.tools.helpers import schedulePatAlgosEndPath\n"
+        self.pythonCfgCode+="schedulePatAlgosEndPath(process)\n"
+
```

Comment:
@slava77 do you find it acceptable to add a dummy EndPath to the PAT purely to drag along the needed Tasks?
@wddgit What alternative implements have you considered and what where their strengths and weaknesses?